anchore_engine/twisted.py

```diff
@@ -346,29 +343,15 @@ def _build_api_service(self):
         )
 
         # Build the main site server
-        site = server.Site(root)
+        twistd_logfile = self.service_config.get(
+            "twistd_logfile", self.global_configuration.get("twistd_logfile", None)
+        )
+        if twistd_logfile:
+            site = server.Site(root, logPath=bytes(twistd_logfile, "utf-8"))
```

Comment:
might be able to unify streams using a log formatter, should look into https://twistedmatrix.com/documents/15.3.0/web/howto/web-in-60/access-logging.html